### PR TITLE
Refactor internationalized document generation

### PR DESCRIPTION
**Pull request by alimakki**
Streisand's current strategy for generating the various internationalized instructions and pages contain a lot of repetitive tasks.

The repetitive pattern of converting markdown to html is similar enough between all roles that utilize them make it an ideal candidate to transitioning them into their own role that can be included when necessary - as such the newly introduced `i18n-docs` role is similar in vein to the `downloand-and-verify` and `certificates` roles. 

Example usage:

```
# loops through instructions templates and generate an index.html and index-fr.html
- include_role:
    name: i18n-docs
  vars:
    vpn_name: "OpenConnect/AnyConnect"
    input_template_name: "instructions"
    i18n_location: "{{ ocserv_gateway_location }}"
```

```
# output file names can be overridden from the default (no file extension)
- include_role:
    name: i18n-docs
  vars:
    vpn_name: "Gateway firewall"
    i18n_location: "/tmp/"
    input_template_name: "firewall"
    output_file_name: "{{ streisand_server_name }}-firewall-information"
```

### CODE REVIEW

File: playbooks/roles/i18n-docs/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+- name: Generate the {{ vpn_name }} Markdown page
```
[the remark this answers is not in the document]
Comment (alimakki) [reply playbooks/roles/i18n-docs/tasks/main.yml]: I ended up having to change this variable to `title` and `name` conflicts with the role name which ends up printing out something like `Generate the i18n-docs markdown page`

File: playbooks/roles/streisand-gateway/tasks/docs.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  vars:
+    vpn_name: "Gateway index"
+    i18n_location: "{{ streisand_gateway_location }}"
+    input_template_name: "index"
+
+- include_role:
+    name: i18n-docs
+  vars:
+    vpn_name: "Gateway instructions"
+    i18n_location: "/tmp/"
```
[the remark this answers is not in the document]
Comment (alimakki) [reply playbooks/roles/streisand-gateway/tasks/docs.yml]: sounds reasonable

**Comment by alimakki** [comment]
@cpu I think this is ready for another look.

File: playbooks/roles/streisand-gateway/vars/main.yml [hunk 3]
```diff
@@ -4,6 +4,7 @@ streisand_gateway_url: "https://{{ streisand_ipv4_address }}"
 streisand_gateway_htpasswd_file: "/etc/nginx/htpasswd"
 streisand_gateway_password_file: "/etc/nginx/gateway-password.txt"
 streisand_gateway_password_localpath: "../{{ streisand_local_directory }}/gateway-password.txt"
+streisand_temp_gateway_path: "/etc/tmpgateway"
```
Comment (cpu) [inline playbooks/roles/streisand-gateway/vars/main.yml]: Hate to be a stickler but how about `/var/local/streisand.gateway`

Comment (alimakki) [reply playbooks/roles/streisand-gateway/vars/main.yml]: done

**Pull request merged**